src/app/(sok)/_components/searchResult/SearchResult.tsx

```diff
@@ -1,25 +1,40 @@
-import React, { useEffect, useRef, useState } from "react";
+import React, { MutableRefObject, ReactElement, useEffect, useRef, useState } from "react";
 import { VStack } from "@navikt/ds-react";
 import FavouritesButton from "@/app/favoritter/_components/FavouritesButton";
 import useQuery from "@/app/(sok)/_components/QueryProvider";
 import { QueryNames } from "@/app/(sok)/_utils/QueryNames";
 import Divider from "@/app/(sok)/_components/searchResult/Divider";
 import { SortByValues } from "@/app/(sok)/_components/searchResult/Sorting";
-import PropTypes from "prop-types";
-import SearchResultItem from "./SearchResultItem";
+import { StillingFraSokeresultatDTO } from "@/app/lib/stillingSoekSchema";
 import { SEARCH_CHUNK_SIZE } from "../../_utils/query";
+import SearchResultItem from "./SearchResultItem";
 
-export default function SearchResult({ searchResult }) {
+interface SearchResultProps {
+    searchResult: { totalAds: number; ads: StillingFraSokeresultatDTO[] };
+}
+
+export default function SearchResult({ searchResult }: SearchResultProps): ReactElement | null {
     const query = useQuery();
     const [isDebug, setIsDebug] = useState(false);
-    const resultsPerPage = query.get(QueryNames.FROM) || SEARCH_CHUNK_SIZE;
-    const totalPages = Math.ceil(searchResult.totalAds / resultsPerPage);
-    const page = query.has(QueryNames.FROM) ? Math.floor(query.get(QueryNames.FROM) / resultsPerPage) + 1 : 1;
-    const searchResultRef = useRef();
+
+    const resultsPerPage: string = query.has(QueryNames.FROM)
+        ? query.get(QueryNames.FROM)!
+        : SEARCH_CHUNK_SIZE.toString();
```

Comment:
Hva med å parse `resultsPerPage` til number her, i stedet for å parse senere i koden?

Reply:
Ja, enig! Det ble bedre 😊